readme.md

```diff
@@ -10,248 +8,408 @@ See also: https://github.com/ga-wdi-lessons/ruby-oop
 
-Here's an example:
-```ruby
-# person7.rb: inheritance
+We've added a `say_age` method to `Person` below to better illustrate how inheritance works in Ruby.
 
+```rb
 class Person
   attr_accessor :name
-  attr_reader :hunger_level
+  attr_reader :age
+  @@people = []
 
-  def initialize(initial_name, initial_hunger_level)
+  def initialize(initial_name, initial_age)
     @name = initial_name
-    @hunger_level = initial_hunger_level
+    @age = initial_age
+    @@people.push(self)
   end
 
-  def introduction
-    puts "Hello, I'm #{name}"
+  def say_name
+    puts "Hi, my name is " + @name + "."
   end
 
-  # Custom setter for hunger_level
-  def hunger_level=(new_hunger_level)
-    if new_hunger_level < 0
-      @hunger_level = 0
-    else
-      @hunger_level = new_hunger_level
-    end
+  def say_age
+    puts "I am #{@age} years old."
   end
 
+  def Person.get_people
+    return @@people
+  end
 end
 
+# Inheritance is indicated using the `<` symbol after the child class' name.
 class LoudPerson < Person
-  def introduction
-    puts "HELLO, I'M #{name.upcase}!!"
+  # LoudPerson's `say_name` overrides that of the `Person` class.
+  def say_name
+    puts "HEY YOU, MY NAME IS #{@name.upcase}!"
   end
 end
 
 jill = Person.new("Jill", 10)
-bob = LoudPerson.new("Bob", 10)
-
-puts jill.introduction
-puts
-
-puts bob.introduction
-
-puts "Bob's name: '#{bob.name}'"
-bob.hunger_level = 5
-puts "Bob hunger level: #{bob.hunger_level}"
+bob = LoudPerson.new("Bob", 20)
+
+# Both `say_age` methods produce the same result since it was not overwritten in the `LoudPerson` class.
+jill.say_age
+# => "I am 10 years old."
+bob.say_age
+# => "I am 20 years old."
+
+# The `say_name` methods produce different results because we overwrote the original in the `LoudPerson` class.
+jill.say_name
+# => "My name is Jill."
+bob.say_name
+# => "HEY YOU, MY NAME IS BOB!"
 ```
 
 ### You Do: Inheritance
 
-- Make a sleepy person, who sleeps through the introduction.
-- Make a baby, who can only say "Dada" and is always hungry (hunger_level never reaches 0)
+* Create a `QuietPerson` class that has their own version of `say_name`.
+* Create a `VeryLoudPerson` class that...
+  1. Shares the same attributes as `Person` and `LoudPerson`.
+  2. Uses the same `say_name` method as `LoudPerson`.
+  3. Has their own loud version of the `say_age` method.
+* Create a `TwoHeadedPerson` class that shares the same methods as `Person` but also has a second name attribute (for the other head).
+
+### You Do: Superheroes
 
+<!-- AM: I'm going to add to this exercise. -->
+
+Clone down [this repo](https://github.com/ga-wdi-exercises/superheros) and follow the instructions in the Readme.
 
 
 ## Visibility
 
+<!-- AM: The placement of this seems kind of weird. Is this covered enough in the previous lesson? Or should I integrate it into the earlier review section? -->
+
```

Comment:
this is gonna go right over most people's heads. would rename `Visibility` to `Bonus`